Dockerfile.plugins

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12
+FROM golang:1.15
 ENV GO111MODULE on
```

Comment:
On Amazon VPN, I got go module download errors that were fixed by adding:

```
RUN go env -w GOPROXY=direct
```

Can you add that

Reply:
added